creator-node/src/services/stateMachineManager/stateMonitoring/index.js

```diff
@@ -132,14 +119,14 @@ class StateMonitoringManager {
   }
 
   /**
-   * Enqueues a job that picks up where the previous failed job left off.
+   * Enqueues a job that starts at a random user.
```

Comment:
why randomize?

Reply:
ideally it would start where the last job left off, but Bull's onError doesn't pass in the previous job's info so there's no way to know where it left off unfortunately

Reply:
ah, got it

Reply:
adding clarifying comment would be good

Reply:
good point - added and merging after tests pass